formats/completed_transaction/frontend/examples/completed_transaction_with_promotion.json

```diff
@@ -0,0 +1,52 @@
+{
+  "details": {
+    "external_related_links": [
+
+    ],
+    "promotion_choice": {
```

Comment:
Can this be `promotion`? The "choice" is made in the publisher, and at save/publish the choice is made.